packages/app-project/stores/Organization.js

```diff
@@ -5,9 +5,11 @@ import numberString from './types/numberString'
 const Organization = types
   .model('Organization', {
     id: types.maybeNull(numberString),
+    display_name: types.optional(types.string, ''),
+    listed: types.optional(types.boolean, false),
     primary_language: types.optional(types.string, 'en'),
     slug: types.optional(types.string, ''),
```

Comment:
This shouldn't be optional. All organisations must have a slug. However, tests fail if the slug is required.

Reply:
Same comment for the project model too.

Reply:
The app breaks if you make slugs required. 🤔